aspnetcore/blazor/components.md

```diff
@@ -116,6 +116,73 @@ The following `ParentComponent` can provide content for rendering the `ChildComp
 
 [!code-cshtml[](common/samples/3.x/BlazorSample/Pages/ParentComponent.razor?name=snippet_ParentComponent&highlight=7-8)]
 
+## Attribute splatting and arbitrary parameters
+
+Components can capture and render additional attributes in addition to the component's declared parameters. Additional attributes can be captured in a dictionary and then *splatted* onto an element when the component is rendered using the `@attributes` Razor directive. This scenario is useful when defining a component that produces a markup element that supports a variety of customizations. For example, it can be tedious to define attributes separately for an `<input>` that supports many parameters.
```

Comment:
@rynowak Should we say here that the dictionary needs to have string keys?

Reply:
I wouldn't mention it at this point. We're still still explaining the concept.